src/CanvasView.tsx

```diff
@@ -144,42 +141,50 @@ export const CanvasView: React.FC = () => {
             RESET
           </Button>
         )}
-        <Menu closeOnSelect={true} placement="top-end">
-          <MenuButton
-            as={IconButton}
-            size="sm"
-            isRound
-            aria-label="More info"
-            marginLeft="auto"
-            variant="secondary"
-            icon={
-              <QuestionOutlineIcon
-                boxSize={6}
-                css={{ '& circle': { display: 'none' } }}
-              />
-            }
-          />
-          <Portal>
-            <MenuList fontSize="sm" padding="0">
-              <MenuItem
-                as={Link}
-                href="https://github.com/statelyai/xstate"
-                target="_blank"
-                rel="noreferrer"
-              >
-                XState version 4.23.0
-              </MenuItem>
-              <MenuItem
-                as={Link}
-                href="https://stately.ai/privacy"
-                target="_blank"
-                rel="noreferrer"
-              >
-                Privacy Policy
-              </MenuItem>
-            </MenuList>
-          </Portal>
-        </Menu>
+        <ButtonGroup size="sm" spacing={2} marginLeft="auto">
```

Comment:
This button group doesn't move together with the panel when it's getting collapsed/expanded - which is IMHO suboptimal since you can see it jumping from one spot to another when the other element is animating.

Reply:
Not a blocker though, right? Something we can take care of in a separate PR.

Moving it with the panel is non-trivial.

Reply:
Yes, not a blocker. To make the implementation easy easier we could try to level up our Framer Motion game.